portals

victorg1991 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPD-16348 Restore portals

This reverts commit a17db4e3d9537c61d729cc2e3769068cb1e5effc.

sandrodw3 committed:
LPS-16348 This is not needed if we restore portals

This reverts commit 1f0eedd42145738b18b87f9825e05467fa7b4ec3.

sandrodw3 committed:
LPD-16348 We need this to be focusable when no item is selected

If not doing this, the left sidebar is the last focusable element, so tabbing after it passes the focus to the browser buttons, and when that happens, we are not able to move the focus manually

sandrodw3 committed:
LPD-16348 Hide breadcrumbs with CSS so it's always present in the DOM

sandrodw3 committed:
LPD-16348 Add FocusManager

sandrodw3 committed:
LPD-16348 Do this always, so it also works when deleting the last fragment

sandrodw3 committed:
LPD-16348 Null active item when deleting with shortcut

victorg1991 committed:
LPD-16348 Don't move the focus if the update was not triggered with tab

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
